src/objects/zcl_abapgit_object_wdyn.clas.abap

```diff
@@ -842,5 +852,14 @@ CLASS zcl_abapgit_object_wdyn IMPLEMENTATION.
     io_xml->add( ig_data = mt_sources
                  iv_name = 'SOURCES' ).
 
+    read table ls_component-comp_metadata-descriptions into ls_description index 1.
+    if sy-subrc = 0.
+      lt_sotr = zcl_abapgit_sotr_handler=>read_sotr_wda( conv #( ls_description-component_name ) ).
```

Comment:
Well, abaplint claims "lt_sotr" does not exists but in fact "conv" does not exist.

Reply:
CONV does not work on 702, it was introduced in v740sp02